Alembic/versions/3df848fa09a8_create_stacks.py

```diff
@@ -0,0 +1,38 @@
+"""create stacks
+
+Revision ID: 3df848fa09a8
+Revises: 99f4fc5a9a7b
+Create Date: 2023-06-22 01:29:52.898285
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3df848fa09a8'
+down_revision = '99f4fc5a9a7b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('stacks',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
```

Comment:
Shouldn't this be a required field? I think making the name field nullable does not make sense. Kindly confirm why this is.

Reply:
will fix this. taught the default was false